gr-filter/include/gnuradio/filter/ival_decimator.h

```diff
@@ -0,0 +1,40 @@
+/* -*- c++ -*- */
+/*
+ * Copyright 2020 Free Software Foundation, Inc.
+ *
+ * This file is part of GNU Radio
+ *
+ * SPDX-License-Identifier: GPL-3.0-or-later
+ *
+ */
+
+#ifndef INCLUDED_IVAL_DECIMATOR_H
+#define INCLUDED_IVAL_DECIMATOR_H
+
+#include <gnuradio/filter/api.h>
+#include <gnuradio/sync_decimator.h>
+
+namespace gr {
+namespace filter {
+
+/*!
+ * \brief Filter-Delay Combination Block.
+ * \ingroup resamplers_blk
+ *
+ * \details
+ * This block provides interleaved char decimation without
```

Comment:
please correct this comment, meaning "`char` or `short`" ... or something that represents the new block data types